src/Sylius/Bundle/PaymentBundle/Resources/config/app/state_machine.yml

```diff
@@ -37,4 +37,4 @@ winzou_state_machine:
                 to: refunded
             void:
                 from: [new, processing]
-                to: void
+                to: unknown
```

Comment:
On separate note, `void` is valid transition `from [authorized] to voided`, personally I would consider `unknown` a thing to fix instead.

Let's think about it for 2.0.